src/parser.cc

```diff
@@ -48,20 +42,15 @@ class Parser {
  public:
   Parser(std::string_view buf, const char* filename, std::vector<Stmt*>* stmts)
       : buf_(buf),
-        state_(ParserState::NOT_AFTER_RULE),
         stmts_(stmts),
         out_stmts_(stmts),
-        num_define_nest_(0),
```

Comment:
strictly speaking there is another change in here :-)

Reply:
Yes, technically, but I figured it's too small to even mention.